rim/views.py

```diff
@@ -138,10 +137,10 @@ class EditEquipmentView(UpdateView):
 
 class ClientView(DetailView):
     template_name = 'rim/client.html'
-    model = Client
+    model = Client 
 
     def get_context_data(self, *args, **kwargs):
         context = super(ClientView, self).get_context_data(*args, **kwargs)
-        context['active'] = context['client'].checkout_set.filter(equipment__latest_checkout__pk=F('pk'))
-        context['previous'] = context['client'].checkout_set.exclude(equipment__latest_checkout__pk=F('pk'))
+        context['active'] = context['client'].checkout_set.select_related('equipment', 'location').filter(equipment__latest_checkout__pk=F('pk'))
```

Comment:
DRY these two lines out by putting `context['client'].checkout_set.select_related('equipment', 'location')` into a variable and calling .filter() or .exclude() on it